### PR TITLE
fixed select in add/edit operation, visual fixes

### PR DESCRIPTION
**Pull request by Granko15**
Select options change according to type of operation in add/edit operation window.
Field with date only shown when any type of lending selected.
Minor visual fixes.

### CODE REVIEW

File: src/resources/views/finances/modals/create_operation.blade.php [hunk 1]
```diff
@@ -4,31 +4,33 @@
...
 
-      <option class="expense_opt" value="">Náklady na služobnú cestu</option>
-      <option class="expense_opt" value="">Malý nákup</option>
-      <option class="expense_opt" value="">Nákup na faktúru</option>
-      <option class="expense_opt" value="">Nákup z Marquetu</option>
-      <option class="expense_opt" value="">Pôžička pre niekoho</option>
+      <option class="expense_opt" value="" style="display: none;">Náklady na služobnú cestu</option>
+      <option class="expense_opt" value="" style="display: none;">Malý nákup</option>
+      <option class="expense_opt" value="" style="display: none;">Nákup na faktúru</option>
+      <option class="expense_opt" value="" style="display: none;">Nákup z Marquetu</option>
```
Comment (k00cur1k) [inline src/resources/views/finances/modals/create_operation.blade.php]: you should create new class, not add so much styles to html code

File: src/resources/views/finances/modals/create_operation.blade.php [hunk 2]
```diff
@@ -4,31 +4,33 @@
...
+      <option class="income_opt" value="return_of_lending" style="display: none;">Splatenie pôžičky od niekoho</option>
 
     </select>
     <input type="text" placeholder="Názov">
     <input type="text" placeholder="Subjekt">
     <input type="text" placeholder="Suma">
     <input type="text" placeholder="Názov">
-    <label>Splatné do:</label><input type="date" placeholder="dd.mm.yyyy">
+    <label class="lending_opt" style="display: none;">Splatné do:</label>
+    <input class="lending_opt" style="display: none;" type="date" placeholder="dd.mm.yyyy">
```
Comment (k00cur1k) [inline src/resources/views/finances/modals/create_operation.blade.php]: same with styles

File: src/resources/views/finances/modals/edit_operation.blade.php [hunk 3]
```diff
@@ -4,32 +4,34 @@
...
 
-      <option class="expense_opt" value="">Náklady na služobnú cestu</option>
-      <option class="expense_opt" value="">Malý nákup</option>
-      <option class="expense_opt" value="">Nákup na faktúru</option>
-      <option class="expense_opt" value="">Nákup z Marquetu</option>
-      <option class="expense_opt" value="">Pôžička pre niekoho</option>
+      <option class="expense_opt" value="" style="display: none;">Náklady na služobnú cestu</option>
+      <option class="expense_opt" value="" style="display: none;">Malý nákup</option>
+      <option class="expense_opt" value="" style="display: none;">Nákup na faktúru</option>
+      <option class="expense_opt" value="" style="display: none;">Nákup z Marquetu</option>
```
Comment (k00cur1k) [inline src/resources/views/finances/modals/edit_operation.blade.php]: same with styles

File: src/resources/views/finances/modals/edit_operation.blade.php [hunk 4]
```diff
@@ -4,32 +4,34 @@
...
+      <option class="income_opt" value="return_of_lending">Splatenie pôžičky od niekoho</option>
 
     </select>
     <input type="text" placeholder="Názov">
     <input type="text" placeholder="Subjekt">
     <input type="text" placeholder="Suma">
     <input type="text" placeholder="Názov">
-    <label>Splatné do:</label><input type="date" placeholder="dd.mm.yyyy">
+    <label class="edit_lending_opt" style="display: none;">Splatné do:</label>
+    <input class="edit_lending_opt" style="display: none;" type="date" placeholder="dd.mm.yyyy">
```
Comment (k00cur1k) [inline src/resources/views/finances/modals/edit_operation.blade.php]: same with styles

File: src/resources/views/finances/modals/operation.blade.php [hunk 5]
```diff
@@ -10,9 +10,7 @@
     <p>Suma:</p>
     <p>Dátum vytvorenia:</p>
     <p>Splatné do:</p>
-    <p>Doklad na stiahnutie:</p>
-    <button type="button">Stiahnuť</button>
-
+    <p href="file:PATH_TO_FILE">Doklad na stiahnutie</p>
```
Comment (k00cur1k) [inline src/resources/views/finances/modals/operation.blade.php]: here should be \<a\> tag, not \<p\>

**Review by k00cur1k — CHANGES_REQUESTED** [review]

File: src/resources/views/finances/modals/create_operation.blade.php (hunk 1 above)
Comment (Granko15) [reply src/resources/views/finances/modals/create_operation.blade.php]: done

File: src/resources/views/finances/modals/create_operation.blade.php (hunk 2 above)
Comment (Granko15) [reply src/resources/views/finances/modals/create_operation.blade.php]: done

File: src/resources/views/finances/modals/edit_operation.blade.php (hunk 3 above)
Comment (Granko15) [reply src/resources/views/finances/modals/edit_operation.blade.php]: done

File: src/resources/views/finances/modals/edit_operation.blade.php (hunk 4 above)
Comment (Granko15) [reply src/resources/views/finances/modals/edit_operation.blade.php]: done

**Review by k00cur1k — APPROVED** [review]
thanks

**Pull request merged**